### DISCUSSION TITLE
Visual Studio Code theme?

### DISCUSSION THREAD

**Discussion by Literally3Bagels** [1amSimp1e/dots#94] [Ideas]
Now I know this is an unreasonable request as why would you do work for someone you don't know when it doesn't effect you, but I use VSC and unfortunately there is not a theme for it matching your "Late Night" theme. If you could be so kind as to make this, that would be incredibly appreciated.

**Comment by 1amSimp1e** [comment]
It's because its an Open Source Project so it's my responsibility to make it better and fix some problems may have cause if someone used mine. It would try and create a Vs Code theme. Thanks for your suggestion
